src/common/app.config.js

```diff
@@ -76,10 +77,426 @@ const appConfig = /* @ngInject */ function (envServiceProvider, $compileProvider
   }
 
   $qProvider.errorOnUnhandledRejections(false)
+
+  $translateProvider.translations('en', {
```

Comment:
I think it's a little unfortunate to have these translations in this file. I would wonder if we could have separate language files that we pulled in instead (e.g. en.json, es.json) or if that would be too non-performant.

Reply:
yeah, not ideal. Also, Brian said he'd like to replace this translation mechanism (angular-translate) with angular-gettext, which Scotty had actually already set up in this project. Brian just didn't realize it until recently.

So we'll do some cleanup. Not sure if it'll happen after peak giving or before.

Reply:
I have some time over the next couple of days with Thanksgiving being a while ago in Canada, I can put the translations into a new branch with the commits I did to translate the way I did reverted and do it again using angular-gettext if that is ok

Reply:
Yeah, that sounds good to me. @Omicron7 do you want to give Moe some direction on this?